src/controllers/v1.controller.ts

```diff
@@ -230,6 +233,16 @@ export class V1Controller {
 
       reqRPCID = parseRPCID(parsedRawData)
 
+      const [blockchainRequest] = this.host.split('.')
+
+      // MEV: Where should this actually live???
+      if (blockchainRequest.toLowerCase() === 'eth-rpc' && isEthSendMethod(parsedRawData)) {
+        const BEVResponse = await handleBEVRequest(this.requestID, parsedRawData, this.publicRPCEthAddress)
+        if (BEVResponse) {
```

Comment:
IF BEVResponse is successful you should still perform existing broadcast logic, 
but with 1 block (10-12sec) delay. being not dependent on the BEV relay.